04.ls/ls.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+COLUMN_COUNT = 3
+
+def main
+  filenames_in_dir = Dir.glob('*')
+  show_file_list(filenames_in_dir, COLUMN_COUNT)
```

Comment:
```suggestion
  show_file_list(filenames_in_dir)
```

定数はどこからでも参照できるので引数で渡さなくても大丈夫です